modules/sdk-core/src/bitgo/keychain/keychains.ts

```diff
@@ -214,29 +214,58 @@ export class Keychains implements IKeychains {
       this.bitgo.setRequestTracer(params.reqId);
     }
 
-    return await this.bitgo
-      .post(this.baseCoin.url('/key'))
-      .send({
-        pub: params.pub,
-        commonPub: params.commonPub,
-        commonKeychain: params.commonKeychain,
-        encryptedPrv: params.encryptedPrv,
-        type: params.type,
-        keyType: params.keyType,
-        source: params.source,
-        provider: params.provider,
-        originalPasscodeEncryptionCode: params.originalPasscodeEncryptionCode,
-        enterprise: params.enterprise,
-        derivedFromParentWithSeed: params.derivedFromParentWithSeed,
-        disableKRSEmail: params.disableKRSEmail,
-        krsSpecific: params.krsSpecific,
-        keyShares: params.keyShares,
-        userGPGPublicKey: params.userGPGPublicKey,
-        backupGPGPublicKey: params.backupGPGPublicKey,
-        algoUsed: params.algoUsed,
-        isDistributedCustody: params.isDistributedCustody,
-      })
-      .result();
+    if (params.reducedEncryptedPrv) {
```

Comment:
Kind of weird to pass this in only to pass it back as return?

Reply:
fixed